BaaServiceKit-Example/BaaServiceKit-Example/BlockchainServiceProvider.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+import BaaServiceKit
+
+final class ServiceProvider {
+    static let shared = ServiceProvider()
+
+    let blockchainService = try! BaaService()
```

Comment:
Its a little bit weird to me to use `try!`. If it always exist, why don't we handle this `try!` within the framework?